instrumentation/hibernate/hibernate-4.3/javaagent/hibernate-4.3-javaagent.gradle

```diff
@@ -9,8 +10,23 @@ muzzle {
   }
 }
 
+testSets {
+  version5Test {
+    dirName = 'test'
+  }
+  version6Test {
+    dirName = 'test'
+  }
+
+  latestDepTest {
```

Comment:
You should be able to use `latestDepTestLibrary`, setting to 5.+ to avoid the extra source set. Does it not work?

Reply:
Unfortunately no, with `-PtestLatestDeps=true` fails with
```
Execution failed for task ':instrumentation:hibernate:hibernate-4.3:javaagent:compileVersion6TestJava'.
> Could not resolve all files for configuration ':instrumentation:hibernate:hibernate-4.3:javaagent:version6TestCompileClasspath'.
   > Could not resolve org.hibernate:hibernate-core:+.
     Required by:
         project :instrumentation:hibernate:hibernate-4.3:javaagent
      > Cannot find a version of 'org.hibernate:hibernate-core' that satisfies the version constraints:
           Dependency path 'io.opentelemetry.hibernate-4.3:javaagent:0.16.0-SNAPSHOT' --> 'org.hibernate:hibernate-core:+'
           Dependency path 'io.opentelemetry.hibernate-4.3:javaagent:0.16.0-SNAPSHOT' --> 'org.hibernate:hibernate-core:{strictly 5.+}'
           Dependency path 'io.opentelemetry.hibernate-4.3:javaagent:0.16.0-SNAPSHOT' --> 'org.hibernate:hibernate-core:6.0.0.Alpha6'
```